godtools/App/Share/Views/VideoPlayerView/VideoPlayerView.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  VideoPlayerView.swift
+//  godtools
+//
+//  Created by Robert Eldredge on 11/1/21.
+//  Copyright © 2021 Cru. All rights reserved.
+//
+
+import UIKit
+import youtube_ios_player_helper
+
+class VideoPlayerView: UIViewController {
+    
+    @IBOutlet weak private var youTubeVideoPlayer: YTPlayerView!
+    @IBOutlet weak private var youTubeVideoPlayerLoadingView: UIView!
+    @IBOutlet weak private var youTubeVideoPlayerActivityIndicator: UIActivityIndicatorView!
+    
```

Comment:
Hey @reldredge71 this can be ```let``` ```immutable``` because it is getting set in the initializer of our View and the viewModel will never be mutated after initialization of the View.